releases/release-1.14/README.md

```diff
@@ -78,61 +83,115 @@ The 1.14 release cycle is proposed as follows:
 
 ### Enhancements Freeze
 
-All enhancements going into the release must have an associated issue in the enhancements repo by ***Tuesday, January 29, 2019***. That issue must be in the 1.14 milestone.  SIG "themes" should also be in the release notes draft at this time to prepare for blog posts and release marketing.  Any work the SIG wants publicized needs to be called out to the Enhancements Lead so the Release Team communications lead can work with SIG-PM and the CNCF.
+By **Tuesday January 29, 2019** all enhancements landing in v1.14.0 must:
+
+- have an associated tracking issue in [kubernetes/enhancements] in the
+  [v1.14 milestone][k/e-milestone-v1.14]
+- have an associated KEP merged into [kubernetes/enhancements] with an
+  implementable status, linked to in the description of the tracking issue
```

Comment:
as [discussed in slack](https://kubernetes.slack.com/archives/C2C40FMNF/p1548859924771400), since this was unclear prior to freeze, suggest changing this to:

```
By **Tuesday January 29, 2019** all enhancements landing in v1.14.0 must:
- have an associated tracking issue in [kubernetes/enhancements] in the 
  [v1.14 milestone][k/e-milestone-v1.14]
- have an associated KEP merged or in progress in [kubernetes/enhancements], 
  linked to in the description of the tracking issue

By **Monday, February 4, 2019**, the KEPs associated with all enhancements landing in v1.14.0 
must be merged with an implementable status. This 1.14-specific extension is intended to allow 
time to complete KEPs that were in progress by enhancements freeze. In future releases, the 
intent is to require associated KEPs be merged/implementable by the enhancements freeze date.
```

Reply:
I agree with this change even if I'm not at a keyboard to make it right now

Reply:
added as a third commit in https://github.com/kubernetes/sig-release/pull/482